data/json/items/comestibles/raw_fruit.json

```diff
@@ -100,6 +100,28 @@
     "smoking_result": "dry_fruit",
     "vitamins": [ [ "vitA", 1 ], [ "vitC", 24 ], [ "calcium", 1 ], [ "iron", 2 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "juniper",
+    "name": "handful of junipers",
+    "name_plural": "handful of junipers",
+    "weight": 122,
+    "color": "blue",
+    "spoils_in": "1460 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 2,
+    "healthy": 1,
+    "calories": 56,
```

Comment:
Juniper berries probably shouldn't be edible raw, and if they are should generally dry you out and be unenjoyable. They're a great spice, but not a true fruit

Reply:
I mean you could eat them raw, but yeah. It'd be like eating raw spices. I also reduced the calorie count, because they're really light on them.